src/Animations/babylon.animation.ts

```diff
@@ -314,6 +315,12 @@
                             switch (loopMode) {
                                 case Animation.ANIMATIONLOOPMODE_CYCLE:
                                 case Animation.ANIMATIONLOOPMODE_CONSTANT:
+                                   if(this.transitionFunction)
+                                    {
+                                        var transitionWeight = this.transitionFunction.fadeIn();
+                                        return this.matrixInterpolateFunction(this._target._matrix, startValue, transitionWeight); ////ease from previousAnimation to CurrentFrame with fraction

```

Comment:
I think that this is not required (I may be wrong) but this will be automatically done if you use animatable.enableBlending(0.1).

Does it make sense?